backend/src/chat.gateway.ts

```diff
@@ -122,7 +122,10 @@ export class ChatGateway
         throw new WsException('Invalid Token');
       }
       this.authenticatedSockets.set(client.id, user);
-      return 'SUCCESS';
+      user.memberships.forEach((membership) =>
+        this.handleJoin(client, { channel: membership.channel.id.toString() }),
```

Comment:
```suggestion
  client.join(membership.channel.id.toString());
```
The method handleJoin makes unnecessary database calls in this context and is more adapted when one does not know yet if the user is in the channel.

Reply:
This belongs to another PR and should ot be here, I'll remove it.